modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -547,6 +547,33 @@ Use the Stats API to view your entire organization's usage of Object Store for b
 * Usage per region
 * Usage per store
 
+
+[[osv2-usage-limits]]
+=== Usage Metrics Limits
+
+[%header,cols="10,15,15a"]
+|===
+| Parameter | Description | Values
+| Time series (`period`) | Interval that the query records |
+
+* `1hour`
+* `1day`
+* `1month`	
+
+|Data retention | Length of time that Anypoint MQ retains the data, based on `period`| 
+
+* `period=1hour`: 3 months 
+* `period=1day`: 12 months
+* `period=1month`: 24 months
+
+| Query timeframe | Maximum interval between `startDate` and `endDate` in the query, based on `period` |
```

Comment:
@hannanelson can we add a note here that the API will return HTTP 400 error if the interval between startDate and endDate exceeds the maximum limits we allow?